Sources/Classes/Views/CustomizableButton/CustomizableButtonView.swift

```diff
@@ -175,49 +175,48 @@ open class CustomizableButtonView: UIView {
     }
 
     private func configureSpinnerConstraints() {
-        spinnerView?.translatesAutoresizingMaskIntoConstraints = false
+        guard let spinnerView = spinnerView else {
+            return
+        }
+        spinnerView.translatesAutoresizingMaskIntoConstraints = false
+        var constraints = [NSLayoutConstraint]()
```

Comment:
let

Reply:
there is `if let` above with `else` case, so I've decided to make it var in order not to make `else` cases like `constraints = []`